### PR TITLE
chore(tracer): use hard coded version for the tracer

### PR DESCRIPTION
**Pull request by christophe-papazian**
Following the october 17 meeting in Paris, this is a proposition to replace computation of the version using setuptools with a static file.

## Checklist

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed. If no release note is required, add label `changelog/no-changelog`.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
- [x] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))

## Reviewer Checklist

- [ ] Title is accurate.
- [ ] No unnecessary changes are introduced.
- [ ] Description motivates each change.
- [ ] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [ ] Testing strategy adequately addresses listed risk(s).
- [ ] Change is maintainable (easy to change, telemetry, documentation).
- [ ] Release note makes sense to a user of the library.
- [ ] Reviewer has explicitly acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment.
- [ ] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)
- [ ] If this PR touches code that signs or publishes builds or packages, or handles credentials of any kind, I've requested a review from `@DataDog/security-design-and-guidance`.
- [ ] This PR doesn't touch any of that.

### CODE REVIEW

**Comment by romainkomorndatadog** [comment]
I think I'd go for something where the file containing the version numbers is standalone, and `version.py` reads it.

My thinking here is that it'd be good to make that file something machine-readable that can be read and updated by the release script.

File: pyproject.toml [hunk 1]
```diff
@@ -62,10 +62,6 @@ Documentation = "https://ddtrace.readthedocs.io/en/stable/"
 Homepage = "https://github.com/DataDog/dd-trace-py"
 "Source Code" = "https://github.com/DataDog/dd-trace-py/"
 
-[tool.setuptools_scm]
```
Comment (brettlangdon) [inline pyproject.toml:65]: you can also remove from above under `[build-system]`, and I think it is `setup.py` as well?

Comment (christophe-papazian) [reply pyproject.toml:65]: removing it was breaking things. It looks like either it's used for some thing else, or possibly installing it is also installing other things that we need.

Comment (brettlangdon) [reply pyproject.toml:65]: 🤔

**Pull request closed without merging**